src/data/faq.json

```diff
@@ -14,6 +14,80 @@
+                            "<br>",
+                            "<h3>Certificates</h3>",
+                            "<hr>",
+                            "<b>Which certificates are supported?</b>",
+                            "The Corona-Warn-App supports all three types of EU COVID digital certificates, i.e. vaccination, convalescent and test certificates. You can find more information here: <a href='#eu_dcc_which' target='_blank'>Which EU certificates are supported?</a>",
+                            "<b>How long is my certificate valid?</b>",
+                            "We basically distinguish between the technical expiry date of the certificate and the validity in terms of the duration of protection. The technical expiry date can be found in the certificate details. The duration of protection is currently still the subject of scientific research and depends on factors such as the vaccine administered and age. Your certificate will show as valid during validation if the technical validity has not expired and your certificate complies with the current duration of protection rules, among others. You can find more information here: <a href='#eu_dcc_validity' target='_blank'>How long is my certificate valid?</a>",
+                            "<b>What do I do if incorrect data has been entered in my certificate?</b>",
+                            "When issuing your certificate, please ensure that data such as name, title and date of birth are entered correctly. Typing errors in this data mean that the app cannot assign the certificates to a single person. Have typos in the certificates corrected directly by the issuer. For vaccination certificates, please also ensure that the number of the vaccination is entered correctly. For example, a 3rd vaccination should be entered as '3 of 3' regardless of the vaccine. You can find more information here: ",
+                            "<b>How can I check if my certificate is valid?</b>",
+                            "Before a trip, you can check whether your certificates are valid in a selected country at the time of your trip. The parameters of the certificate are compared with the stored rules. When checking your vaccination certificate, make sure that it is the vaccination certificate that confirms full vaccination protection and make sure that you check your certificate for the correct date of travel. You can find more information here: <a href='#cert_eu_travel' target='_blank'>Is my certificate valid in the country I am traveling to?</a> <a href='#dcc_no_rules' target='_blank'>https://www.coronawarn.app/en/faq/#dcc_no_rules</a>",

```

Comment:
Is the reference to

"<a href='#dcc_no_rules' target='_blank'>https://www.coronawarn.app/en/faq/#dcc_no_rules</a>"

helpful here? "Why does the Corona-Warn-App with version 2.8 or newer show that my certificate couldn't be validated although versions 2.6 & 2.7 showed my certificate as valid?"

If yes, then use the title of the FAQ article not the absolute URL reference as visible text.